js/globals.ts

```diff
@@ -58,6 +65,7 @@ window.clearInterval = timers.clearTimer;
 window.console = new consoleTypes.Console(libdeno.print);
 window.setTimeout = timers.setTimeout;
 window.setInterval = timers.setInterval;
+window.location = new url.URL("file:///"); // TODO this shouldn't be necessary.
```

Comment:
I think this would work, once the `Location` is added to `domTypes`.

```suggestion
window.location = undefined as any as domTypes.Location;
```

As it stands, the type library will emit `location: url.URL` which isn't what you really want.

Reply:
Done